docs/backends/stdio-schema.json

```diff
@@ -0,0 +1,103 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Is it better to use https://json-schema.org/draft/2020-12/schema? In https://json-schema.org/specification-links, I see it's latest release version.